lib/rules/web/admin_console/4.6/monitoring.xyaml

```diff
@@ -33,4 +33,96 @@ browse_to_receiver_config:
 set_pagerduty_key:
   params:    
     input_field_id: integration-key
-  action: set_input_value
\ No newline at end of file
+  action: set_input_value
+
+# metering rules
+check_reports_table:
+  action: goto_chargeback_reports_page
+  action: check_report_table_columns
+  action: check_report_data 
+check_report_table_columns:
+  elements:
+  - selector:
+      xpath: //thead//th[@data-label='Name']
+  - selector:
+      xpath: //thead//th[@data-label='Namespace']
+  - selector:
+      xpath: //thead//th[@data-label='Report Query']
+  - selector:
+      xpath: //thead//th[@data-label='Reporting Start']
+  - selector:
+      xpath: //thead//th[@data-label='Reporting End']
+check_report_data:
+  elements:
+  - selector:
+      xpath: //tbody//td[1]//a[contains(@href,'/k8s/ns/<namespace>/metering.openshift.io~v1~Report/<report_name>')]
+  - selector:
+      xpath: //tbody//td[2]//a[contains(@href,'/k8s/cluster/namespaces/<namespace>')]
+  - selector:
+      xpath: //tbody//td[3]//a[contains(@href,'/k8s/ns/<namespace>/metering.openshift.io~v1~ReportQuery/<reportquery_name>')] 
+navigate_to_chargeback_reportquery_page:
+  params:
+    tab_name: Report Queries
+  action: click_tab
+check_reportquery_details:
+  action: goto_one_reportquery_page
+  action: check_reportquery_query_string
+  action: check_reportquery_columns_table
+check_reportquery_query_string:
+  action:
+    if_param:
+      reportquery_name: namespace-cpu-usage
+    ref: check_namespace_cpu_usage_rq_query_string
+check_reportquery_columns_table:
+  action:
+    if_param:
+      reportquery_name: namespace-cpu-usage
+    ref: check_namespace_cpu_usage_rq_columns_table
+check_namespace_cpu_usage_rq_query_string:
+  element:
+    selector:
+      xpath: //code[contains(.,'FROM {| .Report.Inputs.NamespaceCPUUsageReportName')]
+check_namespace_cpu_usage_rq_columns_table:
+  elements:
+  - selector:
+      xpath: //tbody//td[text()='period_start']
+  - selector:
+      xpath: //tbody//td[text()='period_end']
+  - selector:
+      xpath: //tbody//td[text()='namespace']
+  - selector:
+      xpath: //tbody//td[text()='pod_usage_cpu_core_seconds']
+check_usage_report_table:
+  action: goto_one_report_page
+  action:
+    if_param:
+      report_name: namespace-cpu-usage
+    ref: check_namespace_cpu_usage_usage_report_table
+check_namespace_cpu_usage_usage_report_table:
```

Comment:
two "usage" in this rule name?

Reply:
@yanpzhan  Ah yes, the report name is `namespace-cpu-usage` and the table name is `Usage Report`. Do you think I need to change to other name?